### PR TITLE
Fpi 8 reach stable state

### PR DESCRIPTION
**Pull request by broomburgo**

### CODE REVIEW

File: Sources/Coproduct.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+extension Coproduct where A: Equatable, B: Equatable {
+	public static func == (lhs: Coproduct, rhs: Coproduct) -> Bool {
+		return lhs.fold(
+			onLeft: { left in
+				rhs.fold(
+					onLeft: { left == $0 },
+					onRight: { _ in false })
+		},
+			onRight: { left in
+				rhs.fold(
```
Comment (ChrisMyrants) [inline Sources/Coproduct.swift]: this should be called `right`, not `left`

**Review by ChrisMyrants — APPROVED** [review]

File: Sources/Coproduct.swift (hunk 1 above)
Comment (broomburgo) [reply Sources/Coproduct.swift]: done

Comment (broomburgo) [reply Sources/Coproduct.swift]: done

**Review by ChrisMyrants — APPROVED** [review]

**Review by ChrisMyrants — APPROVED** [review]
Can be merged

**Pull request merged**